contracts/value/mod.go

```diff
@@ -0,0 +1,245 @@
+// Package value implements a simple native contract that can store, delete, and
```

Comment:
I'm not sure I understand correctly, but if I do: 

1) I'm a bit confused by the fact that reading data is only useful for logging.

2) It's not possible to list (`display`) values in the store/snapshot: only the ones that have been written within the current execution of the Dela node will be listable. If I understand correctly, it means that two nodes (with perfect uptime) joining at different points in time the ledger might behave differently for the `display` command... do I get this right? If so, that seems very wrong.

Reply:
1. When you send a transaction, there is no notion of "returned value". It works like the following: A client can only send a transaction to the pool, which is a simple operation that returns once the transaction has been submitted. From there, the client can only wait for the leader to include that transaction in a block and, if the block is accepted, that transaction will be executed on each node and update each node's state. The only mean we have to "get  a value" (at least right now) is to display that value upon the transaction's execution.

2. A snapshot only provides Get/Set/Delete operations. So you can only get the values whose keys are known to you. 

> do I get this right?

It depends on the joining mechanism. If we require a joining node to only load the current database, yes, it will not display the same thing as the other nodes. If we make a new node replay the chain then no, it will have the exact same state (`c.index`) as the other nodes, displaying the same result.

Reply:
Re. point (1), I believe I understand the notion of transaction without a return value.
However, my thinking was with respect to contract interactions.  Can contracts call upon other contracts? I assume not yet, but do we want to support that in the future? If so, the output of one might inform the input of another, correct?

Or are you saying that this is so far out into the future, and requiring so many changes, that this would be the least of our worries? 😅

Reply:
> Or are you saying that this is so far out into the future, and requiring so many changes, that this would be the least of our worries? 😅

Yes. The interface is not ready for that.